hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -61,6 +62,8 @@ public class HoodieClusteringConfig extends HoodieConfig {
       "org.apache.hudi.client.clustering.run.strategy.JavaSortAndSizeExecutionStrategy";
   public static final String PLAN_PARTITION_FILTER_MODE =
       "hoodie.clustering.plan.partition.filter.mode";
+  public static final String PLAN_FILTER_MODE =
```

Comment:
Is additional config really necessary? Can we fold this into the existing config? Though I like the name of the new config, both configs serve a common purpose of filtering (in different ways), and lesser the configs the better. If we have to define a new config, then let's rename it to  `hoodie.clustering.plan.filegroup.filter.mode` just to be more explicit and differentiate from the existing partition filter config.